### DISCUSSION TITLE
Discord Link Expired

### DISCUSSION THREAD

**Discussion by PaulRosson** [cross-seed/cross-seed#584] [General]
Trying to get this up and running, having what's probably a common issue so thought I'd turn to the Discord but the invite link appears to have expired. Would it be possible to get a link here in the meantime?

**Comment by zakkarry** [comment]
Which invite link are you referring to specifically, and where?

**Reply by PaulRosson** [reply]
At the top of this page: https://www.cross-seed.org/

**Comment by zakkarry** [comment]
That link has no expiration.

**Comment by PaulRosson** [comment]
Ah it was a Discord issue, no worries
